code/drasil-code/Language/Drasil/Code/Imperative/Helpers.hs

```diff
@@ -1,18 +1,29 @@
-module Language.Drasil.Code.Imperative.Helpers (
+{-# LANGUAGE TupleSections #-}
+
+module Language.Drasil.Code.Imperative.Helpers (Pair(..),
     blank,spc,oneTabbed,oneTab,vertical,verticalComma,verticalNewLine,
     angles,doubleQuoted,doubleQuotedText,capitalize,containsAll,
     makeLiteralNameValid,makeVarNameValid,makeClassNameValid,powerSet,
-    hmap,himap,vicat,vibcat,vmap,vimap,vibmap, reduceLibs
+    hmap,himap,hicat,vicat,vibcat,vmap,vimap,vibmap, reduceLibs, mapPairFst, 
```

Comment:
and a number of these pretty-printing helpers are also in `drasil-printers`. We should (eventually) pull them out.

Reply:
#1477 created so we don't forget.